Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -747,8 +747,9 @@ public Func<HttpClient> HttpClientFactory
         /// </remarks>
         public AvailabilityStrategy AvailabilityStrategy { get; set; }
 
-        /// <summary>
-        /// provides SessionTokenMismatchRetryPolicy optimization through customer supplied region switch hints
+        /// <summary>
+        /// Provides SessionTokenMismatchRetryPolicy optimization through customer supplied region switch hints which guide SDK-internal retry policies on how early to switch retries to a different region. 

```

Comment:
```suggestion
        /// Provides SessionTokenMismatchRetryPolicy optimization through customer supplied region switch hints which guide SDK-internal retry policies on how early to switch retries to a different region. 
```suggestion
        /// Provides SessionTokenMismatchRetryPolicy optimization through customer supplied region switch hints which guide SDK-internal retry policies on how early to fallback to a different region. 

```
```